Apps/Sandcastle/gallery/Image-Based Lighting.html

```diff
@@ -66,13 +66,10 @@
         }
 
         var environmentMapURL =
-          "https://cesium.com/assets/kiara_6_afternoon_2k_ibl.ktx";
+          "../../SampleData/EnvironmentMap/kiara_6_afternoon_2k_ibl.ktx2";
         var modelURL = "../../SampleData/models/Pawns/Pawns.glb";
 
-        // This environment map was processed using Google's Filament project. To process your own:
-        // 1 - Download the Filament release (https://github.com/google/filament/releases).
-        // 2 - Run `cmgen --type=ktx --deploy=/path/to/output /path/to/image.hdr`. Other formats are also supported. Run `cmgen --help` for all options.
-        // 3 - Take the generated coefficients and the KTX file and load them in CesiumJS as shown below.
+        // This environment map was processed using Google's Filament project.
```

Comment:
The environment map looks different than `master`. Is this because the image changed or did the IBL implementation change? The environment map should come with a prefiltered mipmap so that rougher materials produce blurrier reflections. Filament was doing this previously but I'm not sure what the new KTX2 generation process is like.

More background: https://learnopengl.com/PBR/IBL/Specular-IBL

![ibl_prefilter_map](https://user-images.githubusercontent.com/915398/88180411-57e3d400-cbfb-11ea-8fb9-1154615806dc.png)

From the `Image-Based Lighting` sandcastle:

**master**

![before](https://user-images.githubusercontent.com/915398/88179600-2ae2f180-cbfa-11ea-897f-a2ebaf39c2a8.jpg)

**ktx2-integration**

![after](https://user-images.githubusercontent.com/915398/88179601-2b7b8800-cbfa-11ea-8b58-c498baa93c1a.jpg)

Reply:
Also the new image is 4.28MB compared to the previous 2MB. The smaller the better so that we don't bloat the CesiumJS zip release too much.

Reply:
@YoussefV you mentioned that a straight up KTX -> KTX2 conversion failed. How quick would it be to write our own script just for this? I guess this depends on the number of breaking changes between KTX and KTX2.

Reply:
I think it would take a while because the spec has changed a decent amount. It might be worth it to open an issue in the [KTX2 repo](https://github.com/KhronosGroup/KTX-Software) about it if we're 100% confident the original KTX file is valid.

The reason why the KTX2 file is larger is because the mip map is uncompressed, so it unpacks the RGB values.

Reply:
So I was the one who created the KTX2 image, the mip map didn't make the reflections blurrier because it autogenerated the levels without taking into account it was a specular IBL. I think the best approach here is to fix the existing converter since the problem may most likely be that the converter has a bug.

I'll open up an issue there.